### ISSUE TITLE
Cannot read property 'headTags' of undefined

### ISSUE THREAD

**Issue by hackteck** [jantimon/html-webpack-plugin#1284]
`<%= htmlWebpackPlugin.tags.headTags %> broken.`
I figure this out with
``
<%= htmlWebpackPlugin.files.js.map(src => `<script type="text/javascript" src="${src}"></script>`).join("\r\n  ") %>
``

**Comment by scottbedard** [comment]
@hackteck It looks like this feature is only available in the 4.0 beta versions. Judging by this, the property should be available https://github.com/jantimon/html-webpack-plugin/blob/bf017f31e29161bdcafb03cff90fb635ed54438a/index.js#L998-L1002

I ran into this problem because vue-cli is relying on 3.2, which doesn't appear to have the property https://github.com/jantimon/html-webpack-plugin/blob/e2c6990e94b298ff66bcd885c9a03a78221479f6/index.js#L722-L725

Does anyone know if it is possible to access these prepared tags in 3.2? I looked for them using the `templateParameters` function, but so far no luck.
